arm-sql/2014-04-01/swagger/dataMasking.json

```diff
@@ -0,0 +1,460 @@
+                        "schema": {
+                            "$ref": "#/definitions/DataMaskingPolicy"
+                        }
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/dataMaskingPolicies/Default/rules/{dataMaskingRule}": {
+            "put": {
+                "tags": [
+                    "DataMasking"
+                ],
+                "operationId": "DataMasking_CreateOrUpdateRule",
+                "description": "Creates or updates an Azure SQL Server Firewall rule.",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ServerNameParameter"
+                    },
+                    {
+                        "name": "databaseName",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "description": "The name of the Azure SQL Database for which the data masking rule applies."
+                    },
+                    {
+                        "name": "dataMaskingRule",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "description": "The name of the Azure SQL Database data masking rule."
+                    },
+                    {
+                        "name": "parameters",
+                        "in": "body",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/DataMaskingRule"
+                        },
+                        "description": "The required parameters for createing or updating a data masking rule."
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "",
+                        "schema": {
+                            "$ref": "#/definitions/DataMaskingRule"
+                        }
+                    },
+                    "201": {
+                        "description": "",
+                        "schema": {
+                            "$ref": "#/definitions/DataMaskingRule"
+                        }
+                    }
+                }
+            },
+            "delete": {
+                "tags": [
+                    "DataMasking"
+                ],
+                "operationId": "DataMasking_DeleteRule",
```

Comment:
I would prefer to keep all DataMasking APIs in the DataMasking operation group rather than giving each its own operation group.

---
In reply to: [80148755](https://github.com/Azure/azure-rest-api-specs/pull/555#discussion_r80148755) [](ancestors = 80148755)